<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputIntToStringDialog</class>
 <widget class="QDialog" name="InputIntToStringDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>279</width>
    <height>93</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>279</width>
    <height>93</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>279</width>
    <height>93</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Elevation layers</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lLayers">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QComboBox" name="cbLayers">
     <property name="minimumSize">
      <size>
       <width>176</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>181</width>
       <height>23</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="isLayers">
     <property name="maximum">
      <number>100</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>InputIntToStringDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>InputIntToStringDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>274</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbLayers</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>InputIntToStringDialog</receiver>
   <slot>changeLayer(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>121</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>7</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>changeLayer(QString)</slot>
  <slot>changeZValue(double)</slot>
 </slots>
</ui>
